sql/moz-fx-data-shared-prod/messaging_system_derived/normalized_onboarding_events/view.sql

```diff
@@ -0,0 +1,28 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.messaging_system_derived.normalized_onboarding_events`
+AS
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  submission_timestamp AS timestamp,
+  version AS app_version,
+  message_id AS category,
```

Comment:
This is... not exactly correct - message_id is not a category of events. But I do know that this is the primary dimension UJET needs to filter on, so it should be top-level (not an event property); and we can rename it in the Looker view without changing these underlying ETL queries.

Reply:
```suggestion
  -- message_id is not a category of events, but it _is_ the primary dimension
  -- UJET needs to filter on, so it should be top-level (not an event property); 
  -- we can rename it in the Looker view.
  message_id AS category,
```